apps/web/src/app/teams/[id]/activity/page.tsx

```
@@ -0,0 +1,100 @@
+                  <Stack key={key}>
+                    <Divider
+                      label={
+                        <>
+                          <Group gap={5} align="center">
+                            {title}
+                            <Badge variant="default" size="xs">
+                              {pullRequests.length}
+                            </Badge>
+                          </Group>
+                        </>
+                      }
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)